src/interface/src/app/plan/project-areas-metrics/chart-data.ts

```diff
@@ -4,4 +4,5 @@ export interface ChartData {
   key: string;
   values: number[];
   metric_layer: string;
+  isPrimary: boolean;
 }
```

Comment:
should we use snake case to match the above?

Reply:
good call!